portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,8 +69,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
       String content = (String) entity.getProperty("content");
+      String score = (String) entity.getProperty("sentimentScore");
+      int scoreInt = Integer.parseInt(score);
 
-      Comment comment = new Comment(id, content);
+      Comment comment = new Comment(id, content, scoreInt);
```

Comment:
Are there some comments in your old Datastore DB that don't have a score attached? How does the code handle this?